### PR TITLE
Add a 'pending' option to the dropdown

### PR DESCRIPTION
**Pull request by Silvan-WMDE**
Bug: [T294573](https://phabricator.wikimedia.org/T294573)

### CODE REVIEW

**Review by guergana — DISMISSED** [review]
Works great, thanks

**Comment by Silvan-WMDE** [comment]
Sorry, the PR needed a rebase after #236 was merged, and somehow the approval didn't survive that.

**Review by guergana — APPROVED** [review]
Re-approving after rebase

**Pull request merged**
